File: Demo/DemoCSharp/Collections/Element.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Demo.Collections
{
	class Element
	{
		public object Data { get; set; }

		public Element(object data)
		{
			this.Data = data;
		}

		public override string ToString()
		{
			return "My element";
		}
	}
}

File: Demo/DemoVisualBasic/Collections/Element.vb

Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks

Namespace Demo.Collections
	Class Element
		Public Property Data As Object

		Public Sub New(data As Object)
			Me.Data = data
		End Sub

		Public Overrides Function ToString() As String
			Return "My element"
		End Function
	End Class
End Namespace
